src/components/Cover/index.js

```diff
@@ -208,4 +193,6 @@ Cover.propTypes = {
   }),
   caption: PropTypes.string,
   captionUrl: PropTypes.string,
+  blurred: PropTypes.bool,
+  height: PropTypes.oneOf(['full']),
```

Comment:
I think this prop isn't used anymore, right?

If it's still used, then it's super weird that it should be one of one 😄